posthog/migrations/0452_change_replay_team_setting_defaults.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.2.14 on 2024-07-24 17:02
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0451_datawarehousetable_updated_at_and_more"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="team",
+            name="capture_console_log_opt_in",
+            field=models.BooleanField(blank=True, default=True, null=True),
```

Comment:
I don't think that this change affects existing rows... so I don't think we have any reason to worry about table locks here